proxy-node-translator/src/test/java/uk/gov/ida/notification/translator/saml/HubResponseTranslatorTest.java

```diff
@@ -0,0 +1,165 @@
+package uk.gov.ida.notification.translator.saml;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.opensaml.core.config.InitializationService;
+import org.opensaml.saml.saml2.core.Response;
+import uk.gov.ida.notification.VerifySamlInitializer;
+import uk.gov.ida.notification.contracts.HubResponseTranslatorRequest;
+import uk.gov.ida.notification.contracts.verifyserviceprovider.TranslatedHubResponseBuilder;
+import uk.gov.ida.notification.contracts.verifyserviceprovider.TranslatedHubResponseTestAssertions;
+import uk.gov.ida.notification.exceptions.hubresponse.HubResponseTranslationException;
+import uk.gov.ida.saml.core.test.builders.ResponseBuilder;
+
+import java.net.URI;
+
+import static uk.gov.ida.saml.core.test.TestCertificateStrings.STUB_COUNTRY_PUBLIC_PRIMARY_CERT;
+
+public class HubResponseTranslatorTest {
+
+    static {
+        try {
+            InitializationService.initialize();
+            VerifySamlInitializer.init();
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    @Test
+    public void translateShouldReturnResponse() {
```

Comment:
Test method not very clear. It can have a name that better explains what is being tested - do we want to test that it returns a correct response given correct attributes or any response at all?

Reply:
name updated to translateShouldReturnValidResponseWhenIdentityVerified